<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddColumnDialogForm</class>
 <widget class="QDialog" name="AddColumnDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Column to Table</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Column informations</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Table name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_tblNameCBB">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Column data type: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_colTypeCBB">
              <item>
               <property name="text">
                <string>Integer</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Real</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>String</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Date</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Size :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>30</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Column name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_colNameLNE"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="te::qt::widgets::HelpPushButton" name="m_helpBTN">
         <property name="text">
          <string>&amp;Help</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_okBTN">
         <property name="text">
          <string>&amp;Ok</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_cancelBTN">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>te::qt::widgets::HelpPushButton</class>
   <extends>QPushButton</extends>
   <header>terralib/qt/widgets/help/HelpPushButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_colNameLNE</tabstop>
  <tabstop>m_okBTN</tabstop>
  <tabstop>m_helpBTN</tabstop>
  <tabstop>m_tblNameCBB</tabstop>
  <tabstop>m_colTypeCBB</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>m_cancelBTN</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
